### ISSUE TITLE
Will SSDT USB previously created work?

### ISSUE THREAD

**Issue by AlaKru** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#205]
Will the SSDT-UIAC-USB-Z490-VISIOND.aml previously created to be used with usbinjectall work for this new way? Or do I need to create a new SSDT? I am still using the apple thunderbolt monitor right now with its built-in USB ports.

**Comment by AlaKru** [comment]
Or is there an easy way to convert that SSDT into whatever the new one is that is needed? Any help would be appreciated.

**Comment by SchmockLord** [comment]
Of course, you can always disable the new SSDT-USB-Ports-Z490-VisionD.aml and use USBinjectall.kext+SSDT-UIAC.aml  or any of the USBMap.kexts when you are more happy with that.

**Comment by AlaKru** [comment]
I may have written my question a little confusing, But I see, they can not be mixed up. That is what I meant though, I do want to use the kextless version, but I'm just unsure how to create one using my port mapping so that the appleUSBEHC works properly
